core/src/main/java/org/kiwix/kiwixmobile/core/bookmark/BookmarksActivity.java

```diff
@@ -117,7 +123,7 @@ public void onDestroyActionMode(ActionMode mode) {
   };
 
   @Override protected void injection(CoreComponent coreComponent) {
-    coreComponent.inject(this);
+    coreComponent.activityComponentBuilder().build().inject(this);
```

Comment:
you need to pass in the activity, ActivityComponent.Builder has a method for this.

Reply:
```
Caused by: java.lang.IllegalStateException: android.app.Activity must be set
        at dagger.internal.Preconditions.checkBuilderRequirement(Preconditions.java:95)
        at org.kiwix.kiwixmobile.core.di.components.DaggerCoreComponent$CoreActivityComponentBuilder.build(DaggerCoreComponent.java:509)
```
This translates into what I said above.

Reply:
Thank you :)